packages/video_player/video_player/README.md

```diff
@@ -135,4 +135,29 @@ and so on.
 
 To learn about playback speed limitations, see the [`setPlaybackSpeed` method documentation](https://pub.dev/documentation/video_player/latest/video_player/VideoPlayerController/setPlaybackSpeed.html).
 
+### Picture in Picture
+
+#### iOS
+On iOS the picture in picture is linked to the AVPlayerController.
+If you want to enable picture in picture make sure to enable the `audio` capability. (in Xcode it will also say, `audo, airplay & picture in picture`)
```

Comment:
```suggestion
If you want to enable picture in picture make sure to enable the `audio` capability (in Xcode's UI it will say **Audio, AirPlay, and Picture in Picture**).
```

Reply:
Updated the documentation